content/en/docs/concepts/workloads/pods/ephemeral-containers.md

```diff
@@ -76,3 +76,270 @@ you can view processes in other containers.
 
 * Learn how to [debug pods using ephemeral containers](/docs/tasks/debug/debug-application/debug-running-pod/#ephemeral-container).
 
+### Kubernetes Ephemeral Container Security
+Attempting to debug a Pod and realizing that you can't install curl due to 
+security settings has to be a meme at this point. Good security practices 
+are always nice but it often comes at the cost of usability. 
+
+Kubernetes has introduced a  concept called ephemeral containers to deal
+with this problem. Ephemeral containers are temporary containers that can be
+attached after a Pod has been created. You can attach one or more temporary
+container with all the tools you need. While the applications container 
+may have "annoying security features" like a read only file system, an ephemeral 
+container can enjoy all the freedom which writing files entails.
+
+Now that you have the  feature you can start a ephemeral container in any Pod you like  
+
+```shell
+kubectl run ephemeral-demo --image=k8s.gcr.io/pause:3.1 --restart=Never
+kubectl debug -it ephemeral-demo --image=busybox:1.28
+```
+
+Once you gain access to a Shell, you can perform management operations in it.
+The following example manifest contains a Pod where an ephemeral container
+for debugging is included.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: ephemeral-demo
+spec:
+  ...
+  ephemeralContainers:
+  - name: debugger-r59b7
+    image: busybox:1.28
+    imagePullPolicy: IfNotPresent
+    stdin: true
+    tty: true
+```
+                            
+The `ephemeralContainers` field is imilar to `initContainers` and `containers` in that
+it contains a list of containers. However, the `ephemeralContainers` field has certain
+options not available in the other fields. For more details, please refer to the API
+documentation.
+
+One notable feature of `ephemeralContainers` is its ability to configure the security context.
+In theory, a malicious actor could escalate the container's privileges. This risk can
+be effectively mitigated by a robust policy enforcement tool.
```

Comment:
Optional: [Pod Security Admission](https://kubernetes.io/docs/concepts/security/pod-security-admission/) covers ephemeral containers, if you want to point to that.